pkg/scheduler/worker/workerpool.go

```diff
@@ -97,6 +106,11 @@ func (w *Pool) invokeProcessableOpsOnce(ctx context.Context, workerPool *ants.Po
 
 func (w *Pool) startWorkerPool(ctx context.Context) (*ants.PoolWithFunc, error) {
 	w.logger.Infof("Starting worker pool with capacity of %d workers", w.config.PoolSize)
+	w.poolID = uuid.NewString()
```

Comment:
The worker pool has to be coupled to the pod name/id. Each mothership instance has one worker pool and we have to be able to map each instance to the trakced worker pool in our DB otherwise we aren't able to detect orphan worker pool of a pod got killed and had no proper shutdown.